build.sbt

```diff
@@ -868,7 +869,8 @@ lazy val junitTestOutputsJVM =
     .settings(
       commonJUnitTestOutputsSettings,
       libraryDependencies ++= Seq(
-        "com.novocode" % "junit-interface" % "0.11" % "test"
+        "com.novocode" % "junit-interface" % "0.11"   % "test",
+        "junit"        % "junit"           % "4.13.2" % "test"
```

Comment:
Since there are at least two usages of these dependencies, we might think about moving them to some `val JUnitDependencies`, eg and use it instead. This way when we will update versions in the future, they would need edit in only a single place.

Reply:
Good idea. I'll force-push the changes when the CI will finish its work, in case there is anything else I need to fix.